DESCRIPTION

```diff
@@ -29,5 +28,5 @@ Suggests:
     outbreaks
 RoxygenNote: 6.0.1.9000
 URL: https://github.com/reconhub/epiflows
-BugReports: https://github.com/reconhub/epimaps/issues
+BugReports: https://github.com/reconhub/epiflows/issues
```

Comment:
Once we have a first stable version, we'll need to add a website (`pkgdown::build_site()`)

Reply:
Yes, and we will need to make an issue out of this :)